packages/extension-core/src/libs/Analytics.ts

```diff
@@ -1,242 +1,53 @@
-import keyring from "@polkadot/ui-keyring"
+import type { Properties } from "posthog-js"
 import { DEBUG } from "extension-shared"
-import posthog, { Properties } from "posthog-js"
+import posthog from "posthog-js"
 
-import { initPosthog } from "../config/posthog"
-import { db } from "../db"
-import { AccountType } from "../domains/accounts/types"
-import { appStore } from "../domains/app/store.app"
+import { initPosthog, setPosthogOptInPreference } from "../config/posthog"
 import { settingsStore } from "../domains/app/store.settings"
-import { balancePool } from "../domains/balances/pool"
-import { Balance, Balances } from "../domains/balances/types"
-import { getNftCollectionFloorUsd } from "../domains/nfts"
-import { nftsStore$ } from "../domains/nfts/store"
-import { chaindataProvider } from "../rpcs/chaindata"
-import { hasGhostsOfThePast } from "../util/hasGhostsOfThePast"
-import { roundToFirstInteger } from "../util/roundToFirstInteger"
-
-const REPORTING_PERIOD = 24 * 3600 * 1000 // 24 hours
-
-const ensurePosthogPreferences = (useAnalyticsTracking: boolean | undefined) => {
```

Comment:
This was needed to get around some buggy behaviour in Posthog, not sure if safe to remove